src/app/teams/teams.service.ts

```diff
@@ -47,7 +47,7 @@ export class TeamsService {
         switchMap((response: any) => response !== undefined ?
           this.updateTeam(
             { limit: 12, status: 'active', createdDate: this.couchService.datePlaceholder, teamPlanetCode: configuration.code,
-              parentCode: configuration.parentCode, ...team, ...response }
+              parentCode: configuration.parentCode, ...team, ...response, createdBy: userId }
```

Comment:
we can change **createdBy** to **teamLeader** if we want to have leadership transfer